### PR TITLE
Make ui configurable

### PR DESCRIPTION
**Pull request by isabellaio**
# Summary

Allows the configuration of the ui by environment variables.

# Checklist for Definition of Done

Depending on the task some of the following jobs are not relevant. Please check the items or cross them out (`~~`text`~~`).

* [x] Code is technically reviewed by myself
* [x] Code is manually tested
* [x] Pipeline is green
* [x] Automatic tests for new functionality are created
* [x] Documentation is updated
* [x] Branch is rebased and therefore ready to merge
* [x] Git history is clean

### CODE REVIEW

File: arc42/src/07_deployment_view.adoc [hunk 1]
```diff
@@ -93,8 +93,20 @@ a| Set to the format of the QWAC certificate in the `tpp-qwac-certificate` heade
...
-| DB schema name (depends on used DB and configuration)
-| _(not set)_
+| DB schema name (depends on used DB and configuration)                                | _(not set)_
+| SANDBOX_FEATURE_UI_CERTPAGE_ENABLED            | Enables/Disables the certificate page in the ui
+
+Possible values are: _true_ or _false_            | _true_
+| SANDBOX_UI_CONTACTMAILTO                       | Contact email address
+
+ Format: _mailto:YOUR@MAILADRESS.DE_ | psd2@adorsys.de
+| SANDBOX_UI_LOGO                                | URL to customized ui logo           | Points to the default psd2-accelerator logo embedded in the application
```
Comment (jatiim) [inline arc42/src/07_deployment_view.adoc:103]: The logo in the browser tab is not exchanged. Would be good if logo and tab logo are identical.

Comment (atamanroman) [reply arc42/src/07_deployment_view.adoc:103]: Good point. We haven't talked about the favicon. I'll create a new task for that.

File: service/src/main/java/de/adorsys/psd2/sandbox/portal/app/ConfigService.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  public ConfigService(Environment environment) {
+    this.environment = environment;
+  }
+
+  /**
+   * Returns the Config for the portal.
+   *
+   * @return UiConfig
+   */
+  UiConfig getUiConfig() {
```
Comment (atamanroman) [inline service/src/main/java/de/adorsys/psd2/sandbox/portal/app/ConfigService.java]: Could make sense to bind the properties directly to `UiConfig` to be able to validate them. But good enough for now.

File: ui/src/environments/commonEnvironment.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+const contentUrlsDe = {
```
Comment (atamanroman) [inline ui/src/environments/commonEnvironment.ts]: Why do we need the angular environment feature here? These values don't need to change at build time, right?

Comment (wandi34) [reply ui/src/environments/commonEnvironment.ts]: Done

File: service/src/main/java/de/adorsys/psd2/sandbox/portal/app/ConfigService.java (hunk 2 above)
Comment (wandi34) [reply service/src/main/java/de/adorsys/psd2/sandbox/portal/app/ConfigService.java]: Done

**Pull request merged**